analyzer/codechecker_analyzer/analyzers/analyzer_base.py

```diff
@@ -63,6 +81,15 @@ def version_compatible(cls, configured_binary, environ):
         """
         raise NotImplementedError("Subclasses should implement this!")
 
+    @classmethod
+    @abstractmethod
+    def version_info(cls):
+        """
+        A subclass should have a version_info method
+        to return with the analyzer's version number.
```

Comment:
What is the expected format? Like this: `0.3.5` or like this: `v1.1.`? Must it be valid as per [`StrictVersion`'s definition](https://epydoc.sourceforge.net/stdlib/distutils.version.StrictVersion-class.html)?